tests/unit/types/arrays/test_search_ops.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+import numpy as np
+
+from jina import Document, DocumentArray
+from jina.types.arrays.memmap import DocumentArrayMemmap
+
+
+@pytest.fixture
+def list_doc_examples():
+    d1 = Document(tags={'city': 'Barcelona', 'phone': 'None'})
+    d2 = Document(tags={'city': 'Berlin', 'phone': '648907348'})
+    d3 = Document(tags={'city': 'Paris', 'phone': 'None'})
+    d4 = Document(tags={'city': 'Brussels', 'phone': 'None'})
+    docs = [d1, d2, d3, d4]
+    return docs
+
+
+def test_single_regex(list_doc_examples, tmpdir):
+    regexes = {'city': r'B.*'}
+    D = DocumentArray(list_doc_examples)
```

Comment:
if possible, do not use capital letters as variables

Reply:
I will do it but I don't feel it's a great idea, capitals for Matrices/Tensors/DocumentArrays make a lot of sense, non capital versions for vectors/documents also make sense. Makes everything easier to read.

Reply:
this is standard programming standards, capitals are reserved for class names and not for vars or instances